test/parallel/test-fs-rmdir-recursive-warns-on-file.js

```diff
@@ -17,5 +17,5 @@ tmpdir.refresh();
   );
   const filePath = path.join(tmpdir.path, 'rmdir-recursive.txt');
   fs.writeFileSync(filePath, '');
-  fs.rmdirSync(filePath, { recursive: true });
+  fs.rmdir(filePath, { recursive: true }, common.mustCall());
```

Comment:
```suggestion
  fs.rmdir(filePath, { recursive: true }, common.mustSucceed());
```

Reply:
@tniessen Shouldn't the linter enforce the use of `mustSucceed` here?

Reply:
No, the linter cannot know whether the function passes an error parameter to the callback or not. But in this case `mustSucceed` is the right choice.